llvm/test/CodeGen/AMDGPU/flat-atomicrmw-fmin.ll

```diff
@@ -14949,85 +17080,139 @@ define <2 x bfloat> @flat_agent_atomic_fmin_ret_v2bf16__offset12b_neg__amdgpu_no
 ; GFX942-NEXT:    v_mov_b32_e32 v7, v0
 ; GFX942-NEXT:    v_lshlrev_b32_e32 v0, 16, v7
 ; GFX942-NEXT:    v_and_b32_e32 v3, 0xffff0000, v7
-; GFX942-NEXT:    v_min_f32_e32 v0, v0, v1
```

Comment:
Strangely, there are places like this that git highlights as a change, but I see nothing really changed...